src/Chirp.Web/CheepService.cs

```diff
@@ -5,7 +5,7 @@ namespace Chirp.Web;
 public interface ICheepService
 {
     public int CheepsPerPage { get; }
-    public List<Cheep> GetCheeps(int pageNumber, string? author = null);
+    public List<Cheep> GetCheeps(int pageNumber, string? author = null, bool authorIsThemselves = false);
```

Comment:
I think we should consider naming it "isAuthor"

Reply:
Yeah, i agree. wonder how the build got in there since mine worked, will have a look